Autopilot/boardfiles/Src/freertos.c

```diff
@@ -77,19 +83,29 @@
 /* Private variables ---------------------------------------------------------*/
 /* USER CODE BEGIN Variables */
 
-/* USER CODE END Variables */
-osThreadId defaultTaskHandle;
+// The period for which each of the threads are called
+static const int PERIOD_ATTITUDEMANAGER_MS = 100;
+static const int PERIOD_PATHMANAGER_MS = 100; //TO CONFIRM
+static const int PERIOD_TELEMETRY_MS = 100; //TO CONFIRM
```

Comment:
niiiice. Quick tip. "TODO" is sortof the universal comment for "I'll have to figure this out later", YOu should use it. That way, in the future, we can just search the project for TODOs and see what little things need to be addressed.